### ISSUE TITLE
Make event notifications actionable 

### ISSUE THREAD

**Issue by lpetkov** [kubeshop/botkube#786]
Add a drop down to each Botkube notification message with a list of relevant actions that:

- [x] are supported by Botkube
- [x] are applicable to the resource type
- [x] the bot has permissions to actually run
- [x] Always return enabled verbs for a given channel regardless the resource (this is what @mszostok already did)
    - Simplified implementation, you can hardcode some verbs for specific resources
    - Populate resources dynamically and attach describe if they have `get`
    - `logs` should be handled only for Pods
    - Use`kubectl api-resources -owide` for verbs (use client-go) if that's good idea
    - Create it as a generic package - sync up with #748 owner, as it will be used there as well
    - Can be worked on in parallel
- [x] Documentation
- [x] Basic telemetry: add `eventDropdown` origin for the executed command
- [x] Integration testing out of scope (as we use legacy Slack)

### Update

- [x] ~Support filtering from the flow 2 - https://www.figma.com/proto/UcaGQ1UiBziOQbNPmSwE0T/grep?page-id=0%3A1&node-id=56%3A443&viewport=163%2C220%2C0.08&scaling=scale-down&starting-point-node-id=56%3A443&show-proto-sidebar=1 - reuse the same modal from #788~ - dropped from the scope
- [x] Add delete confirmation - will be followed-up in https://github.com/kubeshop/botkube/issues/824; for now we'll hide the option
    I'll work on a follow-up PR with a special case for delete. Any kind of minimal extra confirmation works:
    - modal dialog
    - render confirmation button
    - render button right next to it
    
    If that'll be too big effort - filter the "delete" option ouf of this dropdown.

Example:

<img width="703" alt="image" src="https://user-images.githubusercontent.com/31132490/193618055-9011f47b-fcf1-4736-b917-0b76eaf6b481.png">

This will make each notification (or most notifications) _actionable_ and therefore will make the bot easier to use.

## Out of scope

**Stretch**
If `kubectl` is not enabled, whenever we have the technical ability to do so, add a button to enable kubectl under each notification:

![image](https://user-images.githubusercontent.com/31132490/193620964-5f837777-b494-4048-bb19-84d8e1e44661.png)

**Comment by mszostok** [comment]
Filtering should not be via dropdown but inlined:
![image](https://user-images.githubusercontent.com/17568639/194042172-51424dd5-eaab-4431-b239-55893a3122b0.png)

**Comment by pkosiec** [comment]
Update: I will hide the "Delete" option for actionable notifications for now. This doesn't impact the "Actionable messages" feature at all.

I discussed the edge case with @lpetkov (which happens when user explicitly enables the "delete" verb for given K8s resource). Unfortunately I ran out of time - sounds like a rare edge case not worth to invest in right now, as we have different priorities related to Kubecon.

I will create a follow-up task for that and we can see if people really want to delete K8s resources from notifications 🙂
